Sources/SpeziFHIRHealthKit/FHIRResource+LoadAttachements.swift

```diff
@@ -39,35 +39,46 @@ extension FHIRResource {
         if let documentReference = r4Resource as? ModelsR4.DocumentReference {
             for textEncodedAttachement in try await textEncodedAttachements(for: healthKitSample, hkHealthStore: store) {
                 let data = FHIRPrimitive(ModelsR4.Base64Binary(textEncodedAttachement.base64EncodedString))
-                if let content = documentReference.content.first(
-                       where: { $0.attachment.contentType?.value?.string == textEncodedAttachement.identifier }
-                   ),
-                   content.attachment.data == nil {
```

Comment:
I think the simplest diff is just to move this statement in the where clause? Could be a single line change?